x-pack/plugins/stack_alerts/public/alert_types/es_query/expression/search_source_expression_form.tsx

```diff
@@ -63,6 +63,8 @@ const withDebounce = debounce((execute: () => void) => execute(), 500, {
   trailing: true,
 });
 
+const HIDDEN_FILTER_PANEL_OPTIONS: SearchBarProps['hiddenFilterPanelOptions'] = ['pinFilter'];
```

Comment:
@jughosta I think it's fine to create a PR to main excluding this file. seems just tests are missing. So we'd also get CI feedback.